code/modules/organs/external/_external.dm

```diff
@@ -880,12 +880,22 @@ Note that amputating the affected organ does in fact remove the infection from t
 				if(src && istype(loc,/turf))
 					throw_at(get_edge_target_turf(src,pick(GLOB.alldirs)),rand(1,3),30)
 				dir = 2
+
 		if(DROPLIMB_BURN)
 			new /obj/effect/decal/cleanable/ash(get_turf(victim))
+
+			for(var/obj/item/organ/O in internal_organs)
+				if(!O.can_disintegrate())
```

Comment:
Might be worth marking it as Urist code with a comment or something, it's low key enough it might be easy to miss in a merge.